fixtures/_integration.cpp.sem.uast

```diff
@@ -208,6 +176,27 @@
          ],
       },
    ],
-   'Prop_IncludeDirectives': [],
+   'Prop_PreprocStatements': [
+      { '@type': "uast:InlineImport",
+         '@pos': { '@type': "uast:Positions",
+            start: { '@type': "uast:Position",
+               offset: 0,
+               line: 1,
+               col: 1,
+            },
+            end: { '@type': "uast:Position",
+               offset: 19,
+               line: 1,
+               col: 20,
+            },
+         },
+         All: true,
+         Names: [],
+         Path: { '@type': "uast:Identifier",
+            Name: "iostream",
```

Comment:
Will it distinguish `<iostream>` and `"iostream"`?

Reply:
Not in the semantic UAST, in the normal UAST there is a `IsSystem` boolean that distinguist between both.

Reply:
I would suggest prepending `./` if `IsSystem == false`. This will mirror the way how other drivers do it.

Reply:
Another option would be including the quotes or the brackets in the name, but that could give parsing headaches for the user so I'll go with your suggestion.

Reply:
Done!